### ISSUE TITLE
Bug? Nikon N-Log crushed

### ISSUE THREAD

**Issue by JoacimSchwartz** [cameramanben/LUTCalc#22]
When creating a LUT using the N-Log option, to match other cameras or even create a rec709, it seems to crush everything extremely, regardless of which settings are used.

![Skärmavbild 2022-01-12 kl  01 43 50](https://user-images.githubusercontent.com/97571457/149234032-223b9db7-3866-41fd-996f-440698d191af.png)
No LUT applied

![Skärmavbild 2022-01-12 kl  01 43 38](https://user-images.githubusercontent.com/97571457/149234061-16a90197-fecc-403f-bbab-66c0f68aaaeb.png)
![Skärmavbild 2022-01-12 kl  01 44 19](https://user-images.githubusercontent.com/97571457/149234048-330604bb-2f42-49d3-a96c-285e8f5b975a.png)
Settings used.

Using other Gamma alternatives does not create replicate the problem. Am I missing some settings or is this a bug?

**Comment by cameramanben** [comment]
Hi Joacim,

thanks for the heads and it looks to be an easy fix. The Nikon spec uses 10-bit code values in one direction, floating points in the other. LUTCalc is floats all the way and for clarity (oops!) I kept the function as per spec but with some divide by multiply by 1023s.

It seems that two of these divide by 1023s got missed out, but not all, which is why the curve charts and preview look right and the LUTs are going so wrong.

I've made the correction, but I haven't pushed any updates for a while and for some reason I'm having trouble pushing the update to Github. As soon as I've figure out what security setting has changed since the last update I'll upload the change for you to try out.

Thanks,

Ben
